test/widget_test.dart

```diff
@@ -1,7 +1,24 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_template/ui/signin/signin_screen.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+import 'mocks/mock_app.dart';
 
 void main() {
-  testWidgets('Example test', (WidgetTester tester) async {
-    expect(0, 0);
-  });
+  tearDown(rootBundle.clear);
+  setUp(() => GoogleFonts.config.allowRuntimeFetching = false);
```

Comment:
What do you think about creating a common setup function for all widget test pages?